app/controllers/reviews_controller.rb

```diff
@@ -50,23 +54,31 @@ def destroy
     render status: 200, json: @review
   end
 
-  def update_rating
-    size = @restaurant.reviews.length
-    updated = size == 0 ? 0 : @restaurant.reviews.to_a.sum{|review| review.rating} / size
-    @restaurant.rating = updated
-    @restaurant.save
+
+  def render_json (params)
+    if (params[:content].nil?)
+      render status: Rack::Utils.status_code(:ok), json: {}
+    else
+      render json: params[:content]
+    end
   end
 
   private
-    def set_review
-      @review = Review.find(params[:id])
-    end
 
-    def set_restaurant
+  def set_review
+    @review = Review.find(params[:id])
```

Comment:
Why is this needed?
An alternative is to simply call `Review.find(x)` whenever you need to get a review, without storing it to a data member.

We should store stuff on data members when we need it to persist across method calls, and to 'actually store it on the object'.

Reply:
Is this a better practice then using the before_action :set_review, function?